src/plugins/WebIndexPlugin.ts

```diff
@@ -48,7 +49,7 @@ $bundles
         }
 
         let jsTags = bundlePaths.map(bundle =>
-            `<script type="text/javascript" src="${bundle}"></script>`
+            `<script ${opts.async ? 'async' : ''} type="text/javascript" src="${bundle}"></script>`
```

Comment:
I have. yet, 33 of them have failed even before I've made my changes, and that number did not increase after.